### PR TITLE
finished method addDecorateAfter

### PR DESCRIPTION
**Pull request by michaelconnor468**

### CODE REVIEW

File: src/foam/dao/EasyDAO.js [hunk 1]
```diff
@@ -828,6 +828,57 @@ foam.CLASS({
...
+          name: 'decorator',
+          javaType: 'foam.dao.ProxyDAO'
+        },
+        {
+          documentation: 'Decorator in the EasyDAO chain to place in relation to',
+          name: 'location',
+          javaType: 'foam.core.ClassInfo'
+        },
+        {
+          documentation: 'If true, decorator chain placed before the dao instead of inbetween the supplied dao and mdao',
```
Comment (jlhughes) [inline src/foam/dao/EasyDAO.js:847]: Assuming this property came later during development.  Can we rename class to addDecorator and then the 'reverse' property becomes 'before' or 'after'.

Comment (michaelconnor468) [reply src/foam/dao/EasyDAO.js:847]: sure

Comment (michaelconnor468) [reply src/foam/dao/EasyDAO.js:847]: "Can we rename class to addDecorator" can you elaborate a bit on this though; I'm not sure which class to rename

Comment (jlhughes) [reply src/foam/dao/EasyDAO.js:847]: Sorry, that should have read 'method', so addDecoratorAfter to addDecorator

**Review by jlhughes — APPROVED** [review]

**Pull request merged**